2nd Pool Creation: Add Osmosis docs

christopherbrumm opened the pull request.

christopherbrumm committed:
update: add Osmosis pool documentation

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/introduction.md:
```diff
@@ -0,0 +1,54 @@
+- **Storage Provider**: Bundlr
+- **Networks**
+  - [Kaon](https://app.kaon.kyve.network/#/pools/1) (Pool Id: 1)
+  - [Korellia](https://app.korellia.kyve.network/#/pools/24) (Pool Id: 24)
```
Korellia has pool id: 30

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+pull them down:
+
+```bash
+wget https://github.com/KYVENetwork/kyvejs/releases/download/%40kyve%2Fkysor%401.0.0-beta.20/kysor-linux-x64.zip
````
use latest KYSOR version (1.0.0)

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+
+```bash
+./kysor valaccounts create \
+--name 'cosmoshub' \
````
--name 'osmosis'

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+
+```bash
+./kysor valaccounts create \
+--name cosmoshub \
````
--name 'osmosis'

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+
+```bash
+./kysor valaccounts create \
+--name 'cosmoshub' \
````
--name 'osmosis'

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+
+```bash
+./kysor valaccounts create \
+--name cosmoshub \
````
--name 'osmosis'

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
```diff
@@ -0,0 +1,362 @@
+
+If you run with KYSOR and have auto download enabled (which is not recommended for Mainnet and Kaon) you can skip the manual binary installation. If you want to run the binaries with or without KYSOR you have to install them manually.
+
+Depending on the runtime the binary installation differs. For Cosmos Hub we have to install the binary from the
```
Cosmos -> Osmosis

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
```diff
@@ -0,0 +1,362 @@
+If you run with KYSOR and have auto download enabled (which is not recommended for Mainnet and Kaon) you can skip the manual binary installation. If you want to run the binaries with or without KYSOR you have to install them manually.
+
+Depending on the runtime the binary installation differs. For Cosmos Hub we have to install the binary from the
+@kyvejs/tendermint-bsync runtime.
```
@kyvejs/tendermint-bsync -> @kyvejs/tendermint

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+If you want to build the latest version you can skip this step.
+
+```bash
+git checkout tags/@kyvejs/tendermint-bsync@x.x.x -b x.x.x
````
@kyvejs/tendermint-bsync -> @kyvejs/tendermint

troykessler commented on docs/validators/protocol_nodes/pools/osmosis/installation.md:
````diff
@@ -0,0 +1,362 @@
+**INFO**: During the binary build log warnings can occur. You can safely ignore them.
+
+```bash
+cd integrations/tendermint-bsync
````
integrations/tendermint-bsync -> integrations/tendermint

christopherbrumm committed:
chore: Refactoring.

christopherbrumm committed:
chore: update Osmosis Docker container restart policies.

luis-rf committed:
Update run_osmosis_node.md

Add documentation steps to create a docker volume to store the persistent .osmosisd folder

christopherbrumm committed:
Merge pull request #68 from luis-rf/patch-1

Update run_osmosis_node.md

christopherbrumm committed:
chore: update bundles endpoint.
